src/utilities.js

```diff
@@ -2,7 +2,7 @@ import GLib from 'gi://GLib';
 import GObject from 'gi://GObject';
 import Gio from 'gi://Gio';
 
-const ByteArray = imports.byteArray;
+import Gettext from 'gettext';
```

Comment:
I think this is unnecessary now that all translatable text is in either extension.js or prefs.js